webpack.config.js

```diff
@@ -17,102 +18,104 @@ module.exports = (env, argv) => {
+      'style-sync': './ArticleTemplates/assets/scss/style-sync.scss',
+      'style-async': './ArticleTemplates/assets/scss/style-async.scss',
+      'fonts-ios': './ArticleTemplates/assets/scss/fonts-ios.scss',
+      'fonts-android': './ArticleTemplates/assets/scss/fonts-android.scss',
+      'interactive': './ArticleTemplates/assets/scss/interactive.scss',
+    },
+    output: {
+      filename: '[name].js',
+      chunkFilename: '[name].js',
+      path: path.resolve(__dirname, 'ArticleTemplates/assets/build'),
+    },
+    module: {
+      rules: [{
+        test: /\.js$/,
+        exclude: /node_modules/,
+        use: [
+          "babel-loader"
+        ],
       },
-      output: {
-        filename: '[name].js',
-        chunkFilename: '[name].js',
-        path: path.resolve(__dirname, 'ArticleTemplates/assets/build'),
+      {
+        test: /\.js$/,
+        exclude: [/node_modules/],
+        use: ['babel-loader']
       },
-      module: {
-        rules: [{
-          test: /\.js$/,
-          exclude: /node_modules/,
-          use: [
-            "babel-loader"
-          ],
-        },
-        {
-          test: /\.js$/,
-          exclude: [/node_modules/],
-          use: ['babel-loader']
+      {
+        test: /\.js$/,
+        exclude: [
+          /node_modules/,
+          path.resolve(__dirname, "ArticleTemplates/assets/js/bootstraps/"),
+          path.resolve(__dirname, "ArticleTemplates/assets/js/modules/")
+        ],
+        use: ['eslint-loader']
+      },
+      {
+        test: /\.scss$/,
+        use: [{
+          loader: MiniCssExtractPlugin.loader,
         },
         {
-          test: /\.js$/,
-          exclude: [
-            /node_modules/,
-            path.resolve(__dirname, "ArticleTemplates/assets/js/bootstraps/"),
-            path.resolve(__dirname, "ArticleTemplates/assets/js/modules/")
-          ],
-          use: ['eslint-loader']
+          loader: "css-loader",
+          options: {
+            url: false,
+            minimize: true,
+            sourceMap: argv.mode !== 'production',
+          }
         },
         {
-          test: /\.scss$/,
-          use: [{
-            loader: MiniCssExtractPlugin.loader,
-          },
-          {
-            loader: "css-loader",
-              options: {
-                url: false,
-                minimize: true,
-                sourceMap: argv.mode !== 'production',
-              }
-          },
-          {
-            loader: 'sass-loader',
-              options: {
-                sourceMap: argv.mode !== 'production'
-              }
-          }]
+          loader: 'sass-loader',
+          options: {
+            sourceMap: argv.mode !== 'production'
+          }
         }]
+      }]
+    },
+    resolve: {
+      modules: [
+        path.resolve(__dirname, 'ArticleTemplates/assets/js'),
+        "node_modules"
+      ]
+    },
+    plugins: [
+      new LiveReloadPlugin(),
```

Comment:
I wonder what this does for the production build? Do you think the live reload and fixtures are worth keeping? considering we'll hopefully be closing this repo soon

Reply:
I didn't realise the livereload would affect prod! Just in case I'll remove it or keep that in a branch so I can use it for dev-only, as there are a couple more things I'm hoping to do on this repo! Fixtures are definitely ok to delete, I think :)